hyperspy/tests/io/test_emd.py

```diff
@@ -159,7 +177,159 @@ def teardown_method(self, method):
         remove(os.path.join(my_path, 'emd_files', 'example_temp.emd'))
 
 
-if __name__ == '__main__':
-
-    import pytest
-    pytest.main(__name__)
+class TestFeiEMD():
```

Comment:
Let me know if you need small test files for different configurations like SuperXG2Lite and DualX